jwst/extract_1d/soss_extract/pastasoss.py

```diff
@@ -337,7 +349,45 @@ def _find_spectral_order_index(refmodel, order):
     return -1
 
 
-def get_soss_traces(refmodel, pwcpos, order, subarray):
+def get_soss_traces(pwcpos, order, subarray="SUBSTRIP256", refmodel=None):
+    """
+    Get the SOSS traces for a given input model and spectral order.
+
+    Parameters
+    ----------
+    pwcpos : float
+        The pupil wheel position angle provided in the FITS header under keyword PWCPOS.
+        Values are expected to be within +/- 0.5 degrees of the commanded position (245.76 degrees).
```

Comment:
I'm assuming this is in reference to the PWCPOS_BOUNDS above?
```suggestion
        Values are expected to be within +/- 0.25 degrees of the commanded position (245.76 degrees).
```